Scala/Scala.sublime-syntax

```diff
@@ -813,6 +857,7 @@ contexts:
       scope: support.type.scala
     - include: base-type-expression
   single-type-expression:
+    - include: annotation
```

Comment:
I think it would be better to either highlight it as `invalid` or simply remove the highlighting for annotations inside of single-type-expression.